src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/MultiTargetProjectTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.VisualStudio.Razor.IntegrationTests;
+
+public class MultiTargetProjectTests(ITestOutputHelper testOutputHelper) : AbstractRazorEditorTest(testOutputHelper)
+{
+    protected override void PrepareProjectForFirstOpen(string projectFileName)
+    {
+        var sb = new StringBuilder();
+        foreach (var line in File.ReadAllLines(projectFileName))
+        {
+            if (line.Contains("<TargetFramework>"))
+            {
+                sb.AppendLine("<TargetFrameworks>net7.0;net8.0</TargetFrameworks>");
```

Comment:
>net7.0;net8.0 [](http://example.com/codeflow?start=49&length=13)

Nit: Maybe take in TFM as a parameter instead of overriding the entire method?

Reply:
It is a parameter, but adding the "s" in "TargetFrameworks" is annoying :) 

I guess I can make the whole line a parameter